packages/react-components/src/shared/src/checkable.ts

```diff
@@ -1,9 +1,11 @@
 import { ChangeEvent, createContext, useContext } from "react";
 import { isNil } from "lodash";
 
+type CheckableValueType = boolean | string | number | CheckableValueType[]
```

Comment:
CheckableValueType > CheckableValue ?

Reply:
Was looking at

```
type CheckableProps = Omit<CheckableContextType, "checkedValue"> & {
    checked?: boolean;
}

export function useCheckableProps({ value }: { value?: boolean | string | number }): [CheckableProps, boolean] {
```

To be consistant, CheckableProps  -> UseCheckablePropsReturn ? Not sure about our naming conventions here we seem to diverge.

Reply:
done x2